### PR TITLE
Adding section to uninstall gatekeeper

### PR DESCRIPTION
**Pull request by dockerymick**
https://github.com/open-cluster-management/backlog/issues/9760

### CODE REVIEW

File: risk_compliance/create_gatekeeper.adoc [hunk 1]
```diff
@@ -9,6 +9,7 @@ Use the gatekeeper operator policy to install the gatekeeper operator and gateke
 * <<creating-a-gatekeeper-policy-from-the-console,Creating a gatekeeper policy from the console>>
 * <<updating-gatekeeper-operator-policy,Updating gatekeeper operator policy>>
 * <<deleting-gatekeeper-operator-policy,Deleting gatekeeper operator policy>>
+* <<uninstalling-gatekeeper,Uninstalling gatekeeper using the gatekeeper operator policy>>
```
Comment (ycao56) [inline risk_compliance/create_gatekeeper.adoc]: it should be `uninstalling-gatekeeper, Uninstalling gatekeeper policy, gatekeeper and gatekeeper operator`

File: risk_compliance/create_gatekeeper.adoc [hunk 2]
```diff
@@ -152,5 +153,25 @@ kubectl get policy <policy-gatekeeper-operator-name> -n <namespace>
 
 Your gatekeeper operator policy is deleted.
 
+[#uninstalling-gatekeeper]
+== Uninstalling gatekeeper using the gatekeeper operator policy
```
Comment (ycao56) [inline risk_compliance/create_gatekeeper.adoc]: it should be `Uninstalling gatekeeper policy, gatekeeper and gatekeeper operator`

File: risk_compliance/create_gatekeeper.adoc [hunk 3]
```diff
@@ -152,5 +153,25 @@ kubectl get policy <policy-gatekeeper-operator-name> -n <namespace>
...
+
+Complete the following steps to uninstall gatekeeper using the gatekeeper operator policy:
```
Comment (ycao56) [inline risk_compliance/create_gatekeeper.adoc]: `Complete the following steps to uninstall gatekeeper policy, gatekeeper and gatekeeper operator:`

**Comment by dockerymick** [comment]
Thanks for the review @ycao, I have addressed your comments and will continue to merge this PR

**Pull request merged**